app/config/mint/contracts-mainnet.json

```diff
@@ -0,0 +1,7 @@
+{
+	"mint": {
+		"depositAddress": "tb1qjgqhst2hlgjkh36pg6r3hdnz7zvurafe9h5lkx",
+		"bitcoinBroadcastLink": "https://mempool.space/tx/",
+		"suiScanExplorerLink": "https://suiscan.xyz/mainnet/account/"
+	}
+}
```

Comment:
move this file to `config/bitcoin-mainnet.json`

Reply:
it should be `config/mint/bitcoin-mainnet.json`?